features/registration_feature.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Registration page
+  As a customer
+  In order to login
+  I need to be able to register
+
+  Scenario: Visitor sees a form to register on register page
+    Given there is a register page
+    Then I would like to see a form for registration
+    And I would like to see a text field for user name
+    And I would like to see a text field for password
```

Comment:
You can make these two steps as one:
```
I would like to see a text field for "Password"
```
With this setup, the field name can be more dynamic